_sql/hotelify.sql

```diff
@@ -410,7 +507,7 @@ ALTER TABLE `coupon`
   ADD PRIMARY KEY (`id`),
   ADD UNIQUE KEY `id` (`id`),
   ADD KEY `username` (`username`),
-  ADD KEY `type_id` (`type_id`),
+  ADD KEY `type_id` (`coupon_type_id`),
```

Comment:
Reminder to Front-End: Need to change all the usage of `type_id` to be `coupon_type_id`